building/devices/versal_net.rst

```diff
@@ -0,0 +1,303 @@
+.. _versal_net:
+
+###############################
+AMD-Xilinx Versal Net VNX B2197
+###############################
+
+Instructions below show how to run OP-TEE on the Versal Net VNX B2197 development board.
+Details of the Versal Net can be found in the Versal Net Technical Reference Manual.
+
+Prerequisites
+*************
+
+This port requires proprietary binaries available upon request from your AMD-Xilinx FAE:
+
+    - versal-net-bsp folder with:
+        - ksb_castle_peak_b0_optee_20240507.pdi
+
+Configuring and building for Versal Net
+***************************************
+
+Fetching source code
+====================
+
+The default build is mostly automated and follow generic OP-TEE build procedure:
+
+.. code-block:: console
+
+	$ mkdir ~/optee-project
```

Comment:
I'd rather drop the `~` to show that the tree can be anywhere. That's consistent with https://optee.readthedocs.io/en/latest/building/gits/build.html#step-3-get-the-source-code

Reply:
Indeed, good point thanks.